### ISSUE TITLE
On transforming after fit_transform

### ISSUE THREAD

**Issue by asimovidick** [AutoViML/featurewiz#30]
Hi, its me again :)
I tried your new feature of compatibility with scikit-learn following the suggested code where i found the line of transformation over X_test

```python
from featurewiz import FeatureWiz
features = FeatureWiz(corr_limit=0.70, feature_engg='', category_encoders='', 
dask_xgboost_flag=False, nrows=None, verbose=2)
X_train_selected = features.fit_transform(X_train, y_train)
####################################  THIS LINE I'M TALKING ABOUT
X_test_selected = features.transform(X_test)
############################################################
features.features  ### provides the list of selected features ###
```

But I found that what `features.transform(X_test) ` does is "only" filtering `X_test` by the selected features since this is the code inside `FeatureWiz` class:

```python
def transform(self, X):
        return X[self.features]
```

Specifically, what I'm trying to do is:
1) to use `features` (a FeatureWiz object type) to get the completely transformed and filtered dataset according to the selected features
2) save the `features` object
3) to train a X model using the dataset found in step (1)
4) to load the `features` object saved in step (2)
5) in real life to receive an input and transform it using the loaded `features` object in step (4)
6) to feed my model X with the transformed input in step (5)

I just dont know how to complete the step (5) since any input is just filtered and not transformed
I wonder if not I should use the `transform` function of the `My_Groupby_Encoder` class. If that is true, how could I do that?

Thank you so much for your attention to my question
I don't hesitate to say that your work is simply wonderful and useful

**Comment by AutoViML** [comment]
Hi @asimovidick 👍 

If you get the `features.features` object, you already have selected the features from your dataset that are likely to have a strong signal. You can then use this "filter" as you call it to transform those selected features either by one-hot encoding, or label encoding or any other feature transformation you need. You must use the `features.features` filter only to select the strongest features. Don't try to use it like a sklearn transformer since you don't need to select features everytime in a pipeline.

I hope this explanation makes sense.
AutoViML
